src/node/transaction.h

```diff
@@ -14,11 +14,13 @@
  * Broadcast a transaction
  *
  * @param[in]  tx the transaction to broadcast
- * @param[out] &txid the txid of the transaction, if successfully broadcast
  * @param[out] &err_string reference to std::string to fill with error string if available
- * @param[in]  highfee Reject txs with fees higher than this (if 0, accept any fee)
+ * @param[in]  max_tx_fee reject txs with fees higher than this (if 0, accept any fee)
+ * @param[in]  relay flag if both mempool insertion and p2p relay are requested
+ * @param[in]  wait_callback, wait until callbacks have been processed to avoid stale result due to a sequentially RPC.
```

Comment:
87cca293d5b127e43e35d866167893454658eacd

I think a better description/use-case is needed here.

Reply:
I think that the comments here: https://github.com/bitcoin/bitcoin/pull/15713/files#diff-3f841b5d557d579ef74c43ff582b16ebR16 and here: https://github.com/bitcoin/bitcoin/pull/15713/files#diff-bf669d26afe05f397f3f01bad0c1ff0eR52 are sufficient.